src/Core/src/Platform/Windows/CalendarDatePickerExtensions.cs

```diff
@@ -3,8 +3,7 @@
 
 namespace Microsoft.Maui.Platform
 {
-	//TODO make this public on NET8
-	internal static class CalendarDatePickerExtensions
+	public static class CalendarDatePickerExtensions
```

Comment:
Should we be adding extension methods to these primitive types? Less annoying since it is in the Platform namespace, but maybe this should rather be exposed differently?